packages/ui-core/src/components/toggle/toggle.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, ReactElement, useState, useEffect } from 'react';
+
+import {
+  Label,
+  Track,
+  TitleWrapper,
+  HiddenInput,
+} from './toggle.component.styles';
+
+type Props = {
+  children: ReactElement | string;
+  text?: [string, string];
+  width?: number;
```

Comment:
I think we could remove the `width` property and assume that the toggle component always have same width.